project_task_recurring_activity/tests/test_project_recurrence.py

```
@@ -0,0 +1,319 @@
+                    ],
+                }
+            )
+        )
+
+    def test_check_activity_fields(self):
+        with freeze_time("2020-01-01"):
+            form = Form(self.env["project.task"])
+            form.name = "test recurring task"
+            form.description = "my super recurring task"
+            form.project_id = self.project_recurring
+            form.date_deadline = datetime(2020, 2, 1)
```

Ruff (0.11.9)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)